pymeasure/instruments/agilent/agilent33500.py

```diff
@@ -48,6 +46,317 @@
 # string_validator = joined_validators(capitalize_string, strict_discrete_set)
 
 
+class Agilent33500Channel(Channel):
+    """Implementation of a base Agilent 33500 channel"""
+
+    def __init__(self, instrument, id):
```

Comment:
I don't think you need to redefine `__init__`. Right?

Reply:
You are correct. Removed in and tested in https://github.com/pymeasure/pymeasure/pull/763/commits/3e71c1eee00e7ff48596b6418855cf6a67fe831e